pypy3/pypy3.nuspec

```diff
@@ -20,7 +20,7 @@
     <tags>python python3 pypy jit rpython</tags>
     <summary>PyPy - a fast Python 3 interpreter</summary>
     <description>PyPy is a fast, compliant Python 3.6 interpreter written in Python.</description>
-    <releaseNotes>https://pypy.readthedocs.io/en/latest/release-v7.1.0.html</releaseNotes>
+    <releaseNotes>https://pypy.readthedocs.io/en/latest/release-v7.3.1.html</releaseNotes>
```

Comment:
Ah I just noticed this redirects to a different domain: https://doc.pypy.org/en/latest/release-v7.3.1.html

Reply:
Fixed.